.github/workflows/go_tests.yml

```diff
@@ -53,3 +53,7 @@ jobs:
         run: cd sdks/go/pkg/beam && go fmt ./...; git diff-index --quiet HEAD || (echo "Run go fmt before checking in changes" && exit 1)
       - name: Run vet
         run: cd sdks/go/pkg/beam && go vet --copylocks=false --unsafeptr=false ./... || (echo "Run go vet and fix warnings before checking in changes" && exit 1)
+      - uses: dominikh/staticcheck-action@v1.1.0
```

Comment:
The staticcheck docs say 1.1, but it looks like 1.2 is the latest version. We may as well be on the current version.

https://github.com/marketplace/actions/staticcheck

Reply:
Nvm. We should probably punt that until after we move to 1.18 properly.